job-descriptions/product-manager.md

```diff
@@ -0,0 +1,74 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Product Manager
+
+### About us
```

Comment:
```suggestion
## About us
```

Reply:
This is how headings have been done in all the other job listings. If we change it here we should also change on others for consistency. cc @mercadon and @nicksnyder for reasoning on the heading depth.

Reply:
I assume this was done for visual reasons. A `##` heading is rendered with a horizontal line whereas a `###` heading is not. If we want to change this, we should do it across the board in a separate PR.

Reply:
There are several suggestions that would change standard language and formatting in this PR. Let's use this PR to discuss these changes and I will own making a follow up PR to make everything consistent. 

I checked rendering with the `##` heading and it looks good to me, so I will change this here and follow up with a change for all the other job descriptions.